app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -386,17 +384,6 @@ protected void onActivityResult(int requestCode, int resultCode, Intent intent)
                     return;
                 }
                 break;
-            case UPGRADE_APP:
-                if (resultCode == RESULT_CANCELED) {
-                    return;
-                } else if(resultCode == RESULT_OK) {
```

Comment:
@pmates This is definitely capable of causing issues. After an app update
many, many things could have changed about the structure of the app. Home
screen buttons, text localizations, fixture data. I think home screen
translations are likely the most easy-to-replicate thing that would come
up, but don't remember whether there were buggier/more mission critical
components.

-Clayton

On Tue, Nov 17, 2015 at 2:02 PM, Phillip Mates <notifications@github.com>
wrote:

> In app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java
> <https://github.com/dimagi/commcare-odk/pull/821#discussion_r45103526>:
>
> > @@ -386,17 +384,6 @@ protected void onActivityResult(int requestCode, int resultCode, Intent intent)
> >                      return;
> >                  }
> >                  break;
> > -            case UPGRADE_APP:
> > -                if (resultCode == RESULT_CANCELED) {
> > -                    return;
> > -                } else if(resultCode == RESULT_OK) {
>
> I can't remember. @ctsims <https://github.com/ctsims> / @wpride
> <https://github.com/wpride> do either of you foresee any issues coming up
> if we apply an update without logging out and back in?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-odk/pull/821/files#r45103526>.
>